src/standalone/import-export/importTracker.ts

```diff
@@ -93,11 +90,10 @@ export class ImportTracker implements IExtensionSyncActivationService, IDisposab
             this,
             disposables
```

Comment:
@DonJayamanne should it be `this.disposables`?

Reply:
that shouldn't matter, &  it shouldn't cause a recursive
But needs to be changed to be consistent,